### PR TITLE
Added `layergroup`, `daynight`, `track` flags. Added orbit descriptions.

### PR DESCRIPTION
**Pull request by ZachTRice**
- Added `layergroup` parameter to each layer file. This references the folder (group i.e. airs, modis, reference_orbits), to provide an easy way to filter all layers within a specific group. This is used to now filter orbit tracks in some situations.
- Added `daynight` and `track` to orbit layers. This is now used to better output the titles within the Measurement / Sources add modal view.
- Added `description` parameter to orbit layers and added single orbit descriptions. This is the first of the layers to start using per-layer description for output in the info windows and search descriptions.

### CODE REVIEW

File: common/config/wv.json/layers/airs/AIRS_CO_Total_Column_Day.json [hunk 1]
```diff
@@ -7,7 +7,12 @@
             "tags":     "co",
             "group":    "overlays",
             "product":  "AIRX2RET_DAY",
-
+            "layergroup": [
+              "airs"
+            ],
```
Comment (Benjaki2) [inline common/config/wv.json/layers/airs/AIRS_CO_Total_Column_Day.json]: looks like a duplicate

Comment (minniewong) [reply common/config/wv.json/layers/airs/AIRS_CO_Total_Column_Day.json]: duplicate of what?

Comment (ZachTRice) [reply common/config/wv.json/layers/airs/AIRS_CO_Total_Column_Day.json]: I see it. `layergroup`, I had it happen once or twice when I was doing the auto-find and replace.

Comment (minniewong) [reply common/config/wv.json/layers/airs/AIRS_CO_Total_Column_Day.json]: OK - I've never used this review thing so I missed that :)

**Comment by Benjaki2** [comment]
@minniewong  look good to you?

**Pull request merged**